libs.amazon/external/commons-logging-1.1.1-notice.txt

```diff
@@ -0,0 +1,14 @@
+// ------------------------------------------------------------------
+// NOTICE file corresponding to the section 4d of The Apache License,
+// Version 2.0, in this case for Commons Logging
+// ------------------------------------------------------------------
+
+Commons Logging
```

Comment:
Please remove the useless comment in lines 1-5 (I realise, that the original file most probably holds the same text). We should carry the notice, but we also should analyse it for relevant parts and the comment is useless.